CMakeLists.txt

```diff
@@ -72,6 +75,7 @@ add_action_files(DIRECTORY action FILES
 generate_messages(DEPENDENCIES 
     actionlib_msgs 
     std_msgs
+    geometry_msgs
```

Comment:
@JohnAmadeo I was thinking: yes `geometry_msgs` has this `Point` message, but why don't you define your own 2D point so that we do not waste the third dimension and we do not have to use this dependency?